about.html

```diff
@@ -0,0 +1,217 @@
+          <li><a href="#sponsor">Sponsor</a></li>
+          <li><a href="#news">News</a></li>
+          <li><a href="#campaign" id="campaign">CC Campaign</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <!-- Mobile Navigations -->
+    <nav class="mobile-nav">
+      <div id="close">
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          width="24"
+          height="24"
+          viewBox="0 0 24 24"
+          fill="none"
+        >
+          <path
+            fill-rule="evenodd"
+            clip-rule="evenodd"
+            d="M5.29289 5.29289C5.68342 4.90237 6.31658 4.90237 6.70711 5.29289L12 10.5858L17.2929 5.29289C17.6834 4.90237 18.3166 4.90237 18.7071 5.29289C19.0976 5.68342 19.0976 6.31658 18.7071 6.70711L13.4142 12L18.7071 17.2929C19.0976 17.6834 19.0976 18.3166 18.7071 18.7071C18.3166 19.0976 17.6834 19.0976 17.2929 18.7071L12 13.4142L6.70711 18.7071C6.31658 19.0976 5.68342 19.0976 5.29289 18.7071C4.90237 18.3166 4.90237 17.6834 5.29289 17.2929L10.5858 12L5.29289 6.70711C4.90237 6.31658 4.90237 5.68342 5.29289 5.29289Z"
+            fill="white"
+          />
+        </svg>
+      </div>
+      <ul>
+        <li><a href="#about" class="nav-links">About</a></li>
+        <li><a href="#program" class="nav-links">Program</a></li>
+        <li><a href="#join" class="nav-links">Join</a></li>
+        <li><a href="#sponsor" class="nav-links">Sponsor</a></li>
+        <li><a href="#news" class="nav-links">News</a></li>
+        <li>
+          <a href="#campaign" class="nav-links" id="campaign">CC Campaign</a>
+        </li>
+      </ul>
+    </nav>
+
+    <main>
+      <section id="home">
+        <div class="home-header">
+          <h3 class="theme">"Hello! Coding world"</h3>
+          <h1 class="theme">
+            SOFTWARE DEVELOPERS
+            <br />
+            GLOBAL SUMMIT
+            <br />
+            2023
+          </h1>
+        </div>
+        <div class="home-info">
+          <p class="home-paragraph">
+            The Global Summit brings together the community of experts,
+            academics, and activists who comprise the Creative Commons affiliate
+            network in a different country every two years. This year, we hope
+            to expand our invitation list including organizations and
+            individuals.
+          </p>
+          <p class="home-paragraph-2">
+            Please contact us per Email for nay further questions about CC
+            Global Summit 2015! <br />
+            <a href="">doniceubaru1@gmail.com</a>
+          </p>
+        </div>
+      </section>
+
+      <section id="program">
+        <div class="program-header">
+          <h2>
+            Software Developers <br />
+            Global Summit 2023 Logo
+          </h2>
+          <div class="program-underline"></div>
+          <p>
+            The logo of CC Global Summit 2015 was decided through the logo
+            competition from 8, June to 7, July.
+          </p>
```

Comment:
- [ ] KIndly make sure your design match with the template, this fist section should have a backgound image abd a transparent color.

- [ ] Remember that your project should have the same design as the template but you should presonalize you content with a good logic by choosing a topic different to the template one.

Your design | the template
---|---
![image](https://github.com/donice/Microverse-Portal-Capstone/assets/65068771/46b98c12-fd18-4dd7-9551-7323ea0804b4) | ![image](https://github.com/donice/Microverse-Portal-Capstone/assets/65068771/c414ad66-469c-49c7-bf76-a989dac0606b)

### Apply the same review to the About page